### PR TITLE
feat: export ActionBar and ActionBar.Action components

### PR DESCRIPTION
**Pull request by xaviemirmon**
Linked to #461

### CODE REVIEW

**Review by chrisvxd — CHANGES_REQUESTED** [review]
Great stuff! Picked up on a few small bits

File: packages/core/components/ActionBar/ActionBar.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}: {
+  children: ReactNode;
+  onClick: (e: SyntheticEvent) => void;
+}) => (
+  <button className={getClassName("action")} onClick={onClick}>
+    {children}
+  </button>
+);
+
+ActionBar.Action = Action;
```
[the remark this answers is not in the document]
Comment (xaviemirmon) [reply packages/core/components/ActionBar/ActionBar.tsx]: Fixed

File: packages/core/components/DraggableComponent/index.tsx [hunk 2]
```diff
@@ -138,15 +139,14 @@ export const DraggableComponent = ({
...
-                <Copy size={16} />
-              </button>
-              <button className={getClassName("action")} onClick={onDelete}>
-                <Trash size={16} />
-              </button>
+              <ActionBar label={label}>
+                <ActionBar.Action onClick={onDuplicate}>
+                  <Copy size={16} />
+                </ActionBar.Action>
+                <ActionBar.Action onClick={onDuplicate}>
```
[the remark this answers is not in the document]
Comment (xaviemirmon) [reply packages/core/components/DraggableComponent/index.tsx]: Fixed

File: packages/core/components/DraggableComponent/styles.module.css [hunk 3]
```diff
@@ -122,50 +114,4 @@
...
-    color: var(--puck-color-azure-06);
-    transition: none;
-  }
-}
-
-.DraggableComponent-action:active {
-  color: var(--puck-color-azure-07);
-  transition: none;
-}
+}
```
[the remark this answers is not in the document]
Comment (xaviemirmon) [reply packages/core/components/DraggableComponent/styles.module.css]: Caught by prettier now

**Pull request merged**